disnake/integrations.py

```diff
@@ -119,7 +119,7 @@ class Integration(PartialIntegration):
         Whether the integration is currently enabled.
     account: :class:`IntegrationAccount`
         The account linked to this integration.
-    user: :class:`User`
+    user: Optional[:class:`User`]
```

Comment:
versionchanged?

Reply:
The documentation was outdated, it's been optional for a while now.